manifests/h/Helio/HelioWorkstation/3.7/Helio.HelioWorkstation.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Helio.HelioWorkstation
+PackageVersion: "3.7"
```

Comment:
```suggestion
PackageVersion: 2.0
```

Reply:
We'll have upgrade issues if we do this. I'm going to wait on these until I have time to fix the other ones.